lib/get/get-outdated-destination-content.js

```diff
@@ -2,6 +2,7 @@ import Promise from 'bluebird'
 import log from 'npmlog'
 
 const BATCH_CHAR_LIMIT = 1990
+const BATCH_SIZE_LIMIT = 100
```

Comment:
As discussed, customers using shorter custom id's and bigger payloads may still run into an issue here. So we keep the batch size limit of 100 to keep the server response payload small.

Also, I apply now the correct limit via `limit: batches.length`